lib/aiken/math/rational.ak

```diff
@@ -412,17 +377,11 @@ pub fn round(self: Rational) -> Int {
 }
 
 test round_1() {
-  [
-    (round(ratio(10, 7)) == 1)?,
-    (round(ratio(11, 7)) == 2)?,
-    (round(ratio(3, 2)) == 2)?,
-    (round(ratio(5, 2)) == 3)?,
-    (round(ratio(-3, 2)) == -1)?,
-    (round(ratio(-2, 3)) == -1)?,
-    (round(ratio(-10, 7)) == -1)?,
-    (round(ratio(4, 2)) == 2)?,
-  ]
-    |> list.and
+  (round(ratio(10, 7)) == 1)? && (round(ratio(11, 7)) == 2)? && (round(
+    ratio(3, 2),
+  ) == 2)? && (round(ratio(5, 2)) == 3)? && (round(ratio(-3, 2)) == -1)? && (round(
+    ratio(-2, 3),
+  ) == -1)? && (round(ratio(-10, 7)) == -1)? && (round(ratio(4, 2)) == 2)?
```

Comment:
Maybe wait for a version that has the new keywords? Because this looks awful 😅

Reply:
I am not sure to get it 🤔 ? If we release 0.14 then surely we can use it in the stdlib?